debian/rules

```diff
@@ -19,6 +19,12 @@ DEB_CMAKE_EXTRA_FLAGS := \
 	-DWITH_SYSVINIT=ON \
 	-DWITH_SYSTEMD=$(WITH_SYSTEMD)
 
+# Append CMAKE flag with -DLUAJIT_ENABLE_GC64 value. It is required
+# for CI to build Tarantool with GC64 enabled or disabled dynamically.
+ifneq ($(GC64),)
+	DEB_CMAKE_EXTRA_FLAGS += $(GC64)
+endif
```

Comment:
AFAIU, the package name remains the same. Did you check (on a testing repository) that `apt-get install tarantool` and `apt-get install tarantool-gc64` work as expected?

Reply:
Yes, it was tested.

Reply:
I don't see how `apt-get` will distinguish one package from another, but okay: let's lean on testing and not me.